.goreleaser.yml

```diff
@@ -32,14 +32,25 @@ archives:
       - LICENSE
 
 dockers:
-  -
-    goos: linux
+  - goos: linux
     goarch: amd64
     ids:
     - listmonk
     image_templates:
     - "listmonk/listmonk:latest"
     - "listmonk/listmonk:{{ .Tag }}"
+    - "ghcr.io/knadh/{{ .ProjectName }}:latest"
```

Comment:
I made a new organisation `github.com/listmonk` in case listmonk moves to its own org in the future. This can now be changed to `ghcr.io/listmonk/{{ .ProjectName }}`, right? Just like with the DockerHub org.

Reply:
that will require you to move the project under that organization, but yes, it will work.

Reply:
I did some digging and it looks it if my account has permission to write to the org, that's enough. There's no bearing on where the repository is. Would you be able to maybe test this out and confirm? Thanks!